### PR TITLE
[Gifs] Updates on Gifs' image jumping issue 

### PR DESCRIPTION
**Pull request by thyhmdo**
Umbrella issue: https://github.com/carbon-design-system/carbon-website/issues/4048

**Changed**

- Updated new GIFs and images in Components, Patterns, and Designing pages.

### CODE REVIEW

**Comment by alisonjoseph** [comment]
@thyhmdo will take a look, fyi "password" is spelled wrong in that image 😅

**Comment by alisonjoseph** [comment]
@thyhmdo The gif is 1216x1216 and the png is 608x608

File: src/pages/patterns/loading-pattern/images/inline-loading.gif
Comment (alisonjoseph) [inline src/pages/patterns/loading-pattern/images/inline-loading.gif:1]: "pasword" should be "password"

File: src/pages/patterns/forms-pattern/index.mdx [hunk 1]
```diff
@@ -592,9 +592,9 @@ Common user errors include:
 <Column colLg={8}>
 <GifPlayer color='dark'>
 
-![Example of client-side error message](/images/inline-error-message.gif)
+![Example of client-side error message](/images/text-input-error.gif)
```
Comment (laurenmrice) [inline src/pages/patterns/forms-pattern/index.mdx:595]: This gif is not spanning the full 8 columns.

Screenshot 2024-05-22 at 1 39 53 PM

**Comment by laurenmrice** [comment]
Same comments as Alison on that one gif. And then I added a comment above about one of your other gifs. Other than that it looks good to me!

File: src/pages/components/notification/usage.mdx [hunk 2]
```diff
@@ -251,9 +251,9 @@ dismissed.
 <Column colLg={8}>
 <GifPlayer color='dark'>
 
-![Example of toast notification placement](images/05_toast_context_608.gif)
+![Example of toast notification placement](images/notification.gif)
```
Comment (laurenmrice) [inline src/pages/components/notification/usage.mdx:254]: This is not spanning 8 columns on the website

File: src/pages/patterns/forms-pattern/index.mdx (hunk 1 above)
Comment (alisonjoseph) [reply src/pages/patterns/forms-pattern/index.mdx:595]: I think there is a build cache issue where the preview isn't updating, seeing if I can force it to update. I just checked locally though and its displaying correctly now.

**Review by laurenmrice — APPROVED** [review]
Looks great Thy!!!

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**